src/analyses/goto_check.cpp

```diff
@@ -1152,11 +1141,7 @@ void goto_checkt::bounds_check(
     const exprt &pointer=
       to_dereference_expr(ode.root_object()).pointer();
 
-    if_exprt size(
-      dynamic_object(pointer),
-      typecast_exprt(dynamic_size(ns), object_size(pointer).type()),
-      object_size(pointer));
-
+    auto size=object_size(pointer);
```

Comment:
Nit pick: `const` and clang-format spacing

Reply:
ok